code/modules/organs/subtypes/skrell.dm

```diff
@@ -16,3 +16,20 @@
 
 /obj/item/organ/internal/brain/skrell
 	icon_state = "brain_skrell"
+
+/obj/item/storage/internal/skrell
```

Comment:
Currently this implementation doesn't seem to drop the item when the head (and presumably the tails attached) are destroyed.

I think a better way would be to make the storage be inside their head organ, and have the head organ's destroy handle dropping the item within.

Also, this file `code/modules/organs/subtyles/skrell.dm` is for organs -- This is not an organ.